source/adapters/opencl/command_buffer.cpp

```diff
@@ -74,43 +134,82 @@ UR_APIEXPORT ur_result_t UR_APICALL urCommandBufferAppendMemcpyUSMExp(
         *pSyncPointWaitList,
     [[maybe_unused]] ur_exp_command_buffer_sync_point_t *pSyncPoint) {
 
-  cl_adapter::die("Experimental Command-buffer feature is not "
+  cl_adapter::die("Experimental Command-buffer entry point is not "
```

Comment:
Can we remove these die calls? they result in an abort which isn't ideal now that it's highly likely a user could use these entry-points by accident. Returning the error code will be enough for the SYCL runtime to throw an sensible exception that the end user can deal with how they please.